<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractionDialog</class>
 <widget class="QDialog" name="ExtractionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>请选择断面提取模式</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_section">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>单个断面提取</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_sections">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>批量断面提取</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
